generator/lookml.py

```diff
@@ -120,6 +121,9 @@ def _lookml(namespaces, glean_apps, target_dir):
         for view_path in _generate_views(client, view_dir, views, v1_name):
             logging.info(f"    ...Generating {view_path}")
 
+        logging.info("  Generating datagroups")
```

Comment:
Should we use f-string settings for managing the padding? Then we could globally define different indentation levels?
This is what I mean: https://stackoverflow.com/a/5676884

Reply:
Yeah I don't see why not, at the very least we could replace some of these with \t (in a follow-up 😆)